src/api/controllers/verifyEmail.js

```diff
@@ -0,0 +1,41 @@
+import decodejwt from '../../helpers/tokens/decode.token';
+import status from '../../helpers/constants/status.codes';
+import models from '../models/index';
+import error from '../../helpers/error.sender';
+import messages from '../../helpers/constants/error.messages';
+/**
+ * @class
+ */
+export default class {
+  /**
+     * @description verification link controller
+     * @param {*} req
+     * @param {*} res
+     * @returns {*} void
+     */
+  static async verifyEmail(req, res) {
+    const { token } = req.params;
+    const result = decodejwt(token);
+    if (result !== undefined) {
+      const action = await models.User.update({ verified: true }, {
+        where: {
+          email: result.user.email,
+        }
+      });
+      const ZERO = 0;
```

Comment:
@truestbyheart I see what you're doing here, but perhaps there's a better way to check if the action was successful, other than doing action[0]?